src/opm/parser/eclipse/EclipseState/Schedule/Schedule.cpp

```diff
@@ -193,7 +193,7 @@ namespace Opm {
                 handleWGRUPCON(keyword, currentStep);
 
             else if (keyword.name() == "COMPDAT")
-                handleCOMPDAT(keyword, currentStep, grid, eclipseProperties);
+                handleCOMPDAT(parseContext, keyword, currentStep, grid, eclipseProperties);
```

Comment:
The `parseContext` argument is the *least* important argument and should come *last*

Reply:
I placed it as first argument for consistency (there are 3 other function calls that include parseContext, they all have it as their first argument). Otherwise I agree.